docs/internal-documentation/distrib-testing/FreeBSD-Preparation.md

```diff
@@ -18,10 +18,11 @@
    visudo
```

Comment:
Good point. I'm opting to install `sudo`. This way it's easier to have common instructions across operating systems for system testing.